LSP.sublime-settings

```diff
@@ -56,9 +56,22 @@
   // hint: 4
   "show_diagnostics_severity_level": 4,
 
+  // Delay showing diagnostics by this many seconds (as a floating-point
+  // number). The delay will only kick into action when previously there were
+  // no diagnostics in the view. If there were previous diagnostics in the view,
+  // then the delay setting here is ignored and diagnostics are updated
+  // immediately.
+  "diagnostics_delay": 0.0,
+
+  // Add an additional delay when presenting diagnostics, and when the
+  // auto-complete widget is currently visible. See the setting
+  // "diagnostics_delay".
+  "diagnostics_additional_delay_auto_complete": 0.0,
```

Comment:
Not clear to me if this value is treated as an extra on top of `diagnostics_delay` or should generally be bigger than the `diagnostics_delay`.

Also, the defaults here don't match the code in types.py.